modules/oadp-upgrade-from-oadp-data-mover-1-2-0.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="upgrade-from-oadp-data-mover-1-2-0_{context}"]
+= Upgrading from OADP 1.2 Technology Preview Data Mover
+
+{oadp-first} 1.2 Data Mover backups *cannot* be restored with OADP 1.3. To prevent a gap in the data protection of your applications, complete the following steps before upgrading to the OADP 1.3 upgrade:
```

Comment:
```suggestion
{oadp-first} 1.2 Data Mover backups cannot be restored with OADP 1.3. To prevent a gap in the data protection of your applications, complete the following steps before upgrading to the OADP 1.3 upgrade:
```